docs/en_US/Tutorial/Nnictl.rst

```diff
@@ -166,6 +166,10 @@ nnictl resume
      - False
      - 
      - set foreground mode, print log content to terminal
+   * - --experiment_dir, -e
+     - False
```

Comment:
The default value is None?

Reply:
Yes, `nnictl view` will not use `--experiment_dir` by default.